Feature Bars

JakobDaum opened the pull request:
Diese Drei Classen sollen das ein fügen von balken diagramen in die tabelle ermöglichen.
Der Code leuft immer noch nicht Ich habe mich sehr stark (copi/past oft) an dem "star" beispiel orientiert.
PS: 1&1 macht grerade probleme und ichhabe nur sehr sporadish internet.

JakobDaum committed:
Einfügen eines Balkens

die drei klassen sind sehr stark an das "star Rating " beispiel angelent.

alexjanns commented:
Okay, ich hab's mir gerade angeschaut und ein paar Sachen gefixt, werd's gleich pushen. Kompiliert immer noch nicht ohne Fehler, z.B. fehlt bei barCreator eine Funktion sizeHint(). Schau nochmal, dass du bei deinen Klassen auch von den richtigen Basisklassen erbst, dann solltest du das vermutlich noch mit etwas Recherche hinbekommen.
Außerdem musste ich deine ganzen Quelldateien manuell in's Projekt einfügen, hast du die .pro-Datei nicht getracked?

alexjanns committed:
Ein bisschen aufgeräumter

Siehe Kommentar in der Pull Request

Philipp-Huber committed:
Delegate works in tableView

Stripped all unnecessary functions to get delegate to paint something into table.
Furthermore polished model a bit

Philipp-Huber committed:
Prettier bars and refactoring

Bars are now of appropriate length and align with the tabel cells slightly better.
Refactoring for easier manipulation of bar size and overall readability.
Also added property percentageFilled to barCreator, so that it can know how to fill itself.
